python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+                     'will be caught, which may lead to bugs.',
+            'W0705': '\'except\' blocks are analyzed top to bottom, so if we try to catch the same '
+                     'exception multiple times, only the first \'except\' block for the exception '
+                     'will be reached.',
+            'W0706': 'If the \'except\' block uses \'raise\' as it\'s first or only operator, it '
+                     'will just raise back the exception immediately.',
+            'W0711': '\'except\' can catch multiple exceptions, but they have to be passed in as a '
+                     'tuple. Your exception to catch is a result of a binary operation (%s), '
+                     'which will result in only one of the exceptions being caught by the  \'except\' block.',
+            'W0716': '%s is not a valid operation on exceptions.'
+        }
+    },
+    'pylint.checkers.format': {
+        'FormatChecker': {
+            'W0301': '\';\' in Python is used to put multiple statements on a single line, so this '
+                     'semicolon is unnecessary. In general, there is no good reason to ever use a \';\' '
+                     'in Python.',
+            'W0312': 'This code is indented with both %ss and %ss. You should stick to just one, '
+                     'but note that spaces are preferred.',
+            'C0301': 'This line is %s characters long, exceeding our default limit of %s characters.',
+            'C0303': 'Trailing whitespace is poor formatting, and should be removed.',
+            'C0304': 'It\'s proper practice to have the final line in your file be a newline, '
+                     'which it is not.',
+            'C0321': 'Writing multiple statements on a single line is discouraged by PEP8.',
+            'C0330': 'Wrong %s indentation%s%s.\n%s%s. This error occured because of an '
+                     'inconsistent use of number of spaces for indentation.'
+        }
+    },
+    'pylint.checkers.imports': {
+        'ImportsChecker': {
+            'E0401': 'Unable to import %s. Check the spelling of the module name, or '
+                     'whether the module is in the correct directory.',
+            'W0401': 'Wildcard imports add all objects from the import module into the '
+                     'global namespace, which may cause problems. You should only '
+                     'import from %s what you need.',
+            'W0404': 'Module %r was reimported on line %s. A module should not be '
+                     'imported more than once.',
+            'W0410': 'Python 2.5 and greater requires \'__future__\' imports to be the '
+                     'first non-docstring statement in the module.',
+            'C0410': 'Different modules should not be imported on a single line. You '
+                     'should import each module on line %s on separate lines.',
+            'C0414': 'Aliasing an import as the name of the package is useless.',
```

Comment:
"useless" is not helpful. Replace this message with "The 'import <module> as <name>" is used to import a module and give it a different name in this file. If you don't want to rename the module, simply use "import <module>" instead."